db/log_reader.cc

```diff
@@ -303,8 +303,11 @@ void Reader::UnmarkEOFInternal() {
   }
 
   Slice read_buffer;
+  // TODO: rate limit log reader
+  // Note that this might overcharge SequentialFileReader's internal rate
+  // limiter if priority is not IO_TOTAL (when we reach EOF).
```

Comment:
nit: "... is not IO_TOTAL (when we reach EOF) due to implementation limitations" so that at lest someone (you or me) remembers there's left to do for this.

Reply:
Updated with a more clear TODO.